src/main/java/frc/robot/AutoCommands.java

```diff
@@ -135,4 +154,25 @@ public Command scoreAmp() {
         .deadlineWith(new LEDMarqueeCommand(ledSubsystem, 70, 255, 0, 15, 0.08));
   }
 
+  /**
+   * Command to put the turret in intake position
+   * @return new command
+   */
+
+  public Command babyBird() {
+    return new BabyBirdCommand(
+        turretSubsystem, shooterSubsystem).deadlineWith(new LEDBlinkCommand(ledSubsystem, BABYBIRD_COLOR, 0.1));
+  }
+
+  /**
+   * Comand to shoot a note into the middle of the field
+   * @return new command
+   */
+
+  public Command shootMid() {
+    return new ShootTeleopCommand(
+          drivetrainSubsystem, shooterSubsystem, turretSubsystem, ledSubsystem, desiredX,
+          desiredY, STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3);
```

Comment:
I like how you also put the `LEDBlinkCommand` into `babyBird()`, let's do the same here with the `LEDAlternateCommand`.